relevanceai/operations_new/ops.py

```diff
@@ -515,3 +515,38 @@ def apply_transformers_pipeline(
             output_field=output_field,
         )
         return ops.run(self, filters=filters, select_fields=text_fields)
+
+    def scale(
+        self,
+        vector_fields: List[str],
+        model: Optional[str] = None,
+        alias: Optional[str] = None,
+        model_kwargs: Optional[dict] = None,
+        filters: Optional[list] = None,
+        batched: Optional[bool] = None,
+        chunksize: Optional[int] = None,
+    ):
+
+        from relevanceai.operations_new.scaling.ops import ScaleOps
+
+        model = "standard" if model is None else model
```

Comment:
Why wouldn't you just do `model="standard"` ? Same with all the other args - if they're the default they should be set as default? We only do this for mutable objects like lists and dicts

Reply:
True, was just keeping all optional args this way for consistency. Model IS optional